### ISSUE TITLE
Some <va-accordion-item> components have the same ID as headings on the same page

### ISSUE THREAD

**Issue by laflannery** [department-of-veterans-affairs/va.gov-cms#14910]
Related to previous [accordion ID ticket](https://github.com/department-of-veterans-affairs/va.gov-cms/issues/14747)

## Describe the defect
Some accordions and headings on VAMC Health Service pages have identical IDs, and this is not acceptable for screen readers. For example, on the [Boston VAMC Health services page](https://www.va.gov/boston-health-care/health-services/). 

Functionally, the IDs work correctly for the "on this page" links:
- Clicking the anchor link in the "On this page" component brings the user to the heading in both cases
- Using the URL fragment in the URL and going directly to this deep link, opens the proper accordion in both cases

The issue here is still the accessibility concern: there shouldn't be multiple elements with the same ID.

## To Reproduce
1. Go to the [Boston Health Service page](https://www.va.gov/boston-health-care/health-services/)
2. Inspect the Primary Care heading and note that the ID is `primary-care`
3. Then inspect the Primacy care accordion and note that is has the same id of `primary-care`

## AC / Expected behavior

- [ ] Elements on the same page should not have the same ID, each ID should be unique
- [ ] Accessibility Review

 ## Additional context
This was only found on the Health Services page for now, which is Facilities. However, this could potentially happen anywhere headings and accordions exist. Also, this could potentially be resolved when [this other ticket](https://github.com/department-of-veterans-affairs/va.gov-cms/issues/14909) is resolved so I am hesitant to have both PW and Facilities work on this at the same time?

**Comment by wesrowe** [comment]
Need @randimays to take a look at this when she's back in the office.

**Comment by randimays** [comment]
I'm a bit confused as to why there is a PW and Facilities ticket for two issues that are very similar. Should Facilities own the accordions on the pages for facility information? Or does this fall into the PW bucket?

Either way, we shouldn't have two teams working on this at the same time. It seems we have different flavors of the same issue and we need a streamlined approach to solving them.

**Comment by laflannery** [comment]
@randimays can you link me to the Facilities ticket you are referring to? I thought all the accordion IDs tickets were assigned to PW in order to avoid exactly what you had mentioned (2 teams 1 issue)

**Comment by randimays** [comment]
> This was only found on the Health Services page for now, which is Facilities. However, this could potentially happen anywhere headings and accordions exist. Also, this could potentially be resolved when https://github.com/department-of-veterans-affairs/va.gov-cms/issues/14909 is resolved so I am hesitant to have both PW and Facilities work on this at the same time?

Ohh, I thought based on the last sentence that the ticket you linked here was for Facilities. I didn't look closely. Apologize for the confusion!

**Comment by laflannery** [comment]
Ah yep I can see that. No worries and I definitely want to make sure the proper person/team is working on the correct issues so if you or Wes determine that this should be assigned elsewhere of course just let me know!

**Comment by wesrowe** [comment]
@randimays, one thing that came up in FE refinement: we remembered collectively that Josh built a plugin for content-build that fixes a lot of accordions. But none of us was familiar with how it works, what exactly it does, is it relevant here, etc.

So, note that there is that gap in understanding that we would want to either document clearly somewhere (if it isn't already) and/or knowledge-transfer to the team.

And thank you for looking at this in the broader context of accordions.

**Comment by jilladams** [comment]
@chriskim2311 to add estimate and verify if the other dupe ID ticket in sprint fixed this or if separate work is needed, and then do that work.

**Comment by chriskim2311** [comment]
Change is live on prod, all items in accordions are now unique from headers

https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBZzdIIiwiZXhwIjpudWxsLCJwdXIiOiJibG9iX2lkIn19--7746699f7f9c79e9346a20ab109c2625fa19ec64/Health_Services___VA_Boston_Health_Care___Veterans_Affairs.png
